.github/workflows/h100_lychee.yml

```diff
@@ -0,0 +1,90 @@
+# The jobs show up in the github actions UI like this:
+# name: (1) / (2) / (3)
+# e.g.: github-AT2 / lychee / PR_LYCHEE_HOPPER90_CUDA1180_CUDA_LEFT_RIGHT_REL
+# (1) is set by the `name:` field in at2.yaml
+# (2) is set by the name of this file
+# (3) is set by the job name
+
+name: Reusable H100 workflow
+
+permissions:
+  contents: none
+
+on:
+  workflow_call:
+    inputs:
+      kokkos_version:
+        description: 'The Kokkos Core version to build'
+        default: ''
+        required: true
+        type: string
+
+jobs:
+  PR_HOPPER90_CUDA1180_CUDA_LEFT_RIGHT_REL:
+    name: PR_HOPPER90_CUDA1180_CUDA_LEFT_RIGHT_REL
+    runs-on: [lychee-kk-env-cuda-11.8.0-gcc-11.3.0-latest]
+    
+    steps:
+      - name: nvidia-smi
+        run: nvidia-smi
+
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ inputs.kokkos_version }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
+          cmake -S kokkos -B kokkos/build \
+            -DCMAKE_CXX_COMPILER=$PWD/../bin/nvcc_wrapper \
+            -DCMAKE_CXX_FLAGS= \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=kokkos/install \
+            -DKokkos_ENABLE_CUDA=ON \
+            -DKokkos_ARCH_HOPPER90=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
```

Comment:
Can probably remove on of these `-DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF` but not urgent